src/main/java/org/icgc/argo/program_service/utils/EntityService.java

```diff
@@ -6,11 +6,14 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
Name of this file is kind of misleading, as *Service files are inside services folder.

Reply:
It's under utils, so I think it's fine.